js/counties.js

```diff
@@ -91,9 +86,6 @@ function displayCountyData(error, activeCountyData) {
       .attr('id', function(d) {
         return d.properties.GEOID;
       })
-      .text(function(d) {
-        return d.properties.NAME;
-      })
```

Comment:
I don't remember seeing any text before...was this actually working?

Reply:
It's not working on http://dev-owi.usgs.gov/vizlab/water-use-15/#view=SD&category=total right now, so probably not. I thought .text was supposed to be useful as a quick hovertext option, so I'm a little surprised to see that it wasn't doing anything (can you explain?)...but it was always just a placeholder anyway.

Reply:
I think that this only works if you've appended a text element, but not all elements allow text to be added. E.g. `.append("text").text("hi")` would work, but `.append("circle").text("hi")` does not.

Reply:
ah, that explains it =)